airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/snowflake/test_primary_key_streams/final/airbyte_ctes/TEST_NORMALIZATION/NESTED_STREAM_WITH_COMPLEX_COLUMNS_RESULTING_INTO_LONG_NAMES_PARTITION_AB1.sql

```diff
@@ -6,6 +6,7 @@ select
     _AIRBYTE_NESTED_STREAM_WITH_COMPLEX_COLUMNS_RESULTING_INTO_LONG_NAMES_HASHID,
     get_path(parse_json(PARTITION), '"double_array_data"') as DOUBLE_ARRAY_DATA,
     get_path(parse_json(PARTITION), '"DATA"') as DATA,
+    get_path(parse_json(PARTITION), '"column`_''with""_quotes"') as "column`_'with""_quotes",
```

Comment:
Snowflake is able to parse the json blob just fine. It just has an edge case of doubling the `'` or '"' characters to "escape" them...